src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+function sortList(list) {
+  const children = Array.from(list.children);
+
+  children.sort(sortFunc);
+
+  for (const obj of children) {
+    list.appendChild(obj);
+  }
+}
+
+function sortFunc(a, b) {
+  const aSal = formatSalary(a.dataset.salary);
+  const bSal = formatSalary(b.dataset.salary);
+
+  return bSal - aSal;
+}
+
+function formatSalary(salary) {
+  const copy = salary.slice(1);
```

Comment:
In `formatSalary`, the `slice(1)` method assumes that the salary string always starts with a currency symbol. Ensure this is always the case to avoid incorrect parsing.